engine/src/flutter/lib/ui/window.dart

```diff
@@ -968,6 +969,20 @@ class AccessibilityFeatures {
   /// Only supported on iOS.
   bool get onOffSwitchLabels => _kOnOffSwitchLabelsIndex & _index != 0;
 
+  /// Whether accessibility announcements (like [SemanticsService.announce])
+  /// are allowed on the current platform.
+  ///
+  /// Returns `false` on Android, where platform announcements are deprecated
+  /// by the underlying platform.
+  ///
+  /// Returns `true` on all other platforms (iOS, web, desktop) where such
+  /// announcements are generally supported without discouragement.
+  ///
+  /// Use this flag to conditionally avoid making announcements on Android.
+  // This is an inverted check on _index since there are many more platforms
+  // that support announce whereas don't.
```

Comment:
This phrasing is awkward. How about 

`This index check is inverted (== 0 vs != 0); far more platforms support "announce" than discourage it.`

here and below.